src/actions/auth/resetPassword.ts

```diff
@@ -1,15 +1,22 @@
 'use server'
-
-import { createZodActionError } from '@/actions/error'
+import { getUser } from '@/auth/getUser'
 import { safeServerCall } from '@/actions/safeServerCall'
-import { sendResetPasswordMail } from '@/services/notifications/email/systemMail/resetPassword'
-import { emailValidation } from '@/services/notifications/validation'
 import type { ActionReturn } from '@/actions/Types'
-
+import { emailValidation } from '@/services/notifications/validation'
+import { sendResetPasswordMail } from '@/services/notifications/email/systemMail/resetPassword'
+import { verifyResetPasswordToken } from '@/services/auth/resetPassword'
+import { createZodActionError } from '@/actions/error'
 
 export async function resetPasswordAction(rawdata: FormData): Promise<ActionReturn<string>> {
     const parse = emailValidation.typeValidate(rawdata)
     if (!parse.success) return createZodActionError(parse)
 
     return await safeServerCall(() => sendResetPasswordMail(parse.data.email))
 }
+
+export async function verifyResetPasswordTokenAction(token: string): Promise<ActionReturn<{
+    userId: number
+}>> {
+    getUser({})
```

Comment:
There should be comment for why this is here. Someone will see this in the future and get confused. E.g. `// For some reason the project doesn't build if getUser (or actually VevenAdapter specifically) isn't included for all actions. This is most likely a webpack issue and should probably be fixed in the future, but for now simply including it (even if it doesn't actually do anything) will work.`

Reply:
Well maybe, but this will be refactored soon anyways. Hopefully we will have no problems with the general Action HOF

Reply:
Of course the best solution is to fix the issue, but it doesn't hurt to have a comment as long as this is how it's done. It only takes a few seconds to add.